.changeset/six-windows-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/app': patch
+---
+
+Javascript functions/extensions now support environment variable sustitution.
```

Comment:
Do we need a changeset here? Is not the original PR including a similar one already and not released yet?

Reply:
I added it here because I forgot to add one to the original PR 😬